build.gradle

```diff
@@ -739,6 +739,14 @@ class JDKDetails {
     }
 }
 
+tasks.register("lint") {
```

Comment:
Cool! We should definitely run this on PRs to make sure we are keeping high quality source iteration.